corehq/apps/hqwebapp/templates/hqwebapp/partials/form_list_form.html

```diff
@@ -32,11 +32,12 @@
                          data-bind="css: { error: errors.{{ spec.key }} }">
                         {% if spec.type == "RAW" %}
                             <span data-bind="text: data.{{ spec.key }}"></span>
-                        {% elif spec.type == "TextInput" %}
+                        {% else %}
```

Comment:
this now means it accepts every type of spec? just wanted to make sure that's intended